src/DynamicRupture/Output/Builders/OutputBuilder.hpp

```diff
@@ -72,19 +75,38 @@ class OutputBuilder {
     // allocate Rotation Matrices
     // Note: several receiver can share the same rotation matrix
     size_t nReceiverPoints = outputData.receiverPoints.size();
-    outputData.rotationMatrices.resize(nReceiverPoints);
+    outputData.stressGlbToDipStrikeAligned.resize(nReceiverPoints);
+    outputData.stressFaceAlignedToGlb.resize(nReceiverPoints);
```

Comment:
Do `DipStrike` and `FaceAligned` actually mean something different?

Reply:
I think FaceAligned is normal, u1, u2 with u1 along edge 1 of triangle.
DipStrike is normal, strike, dip.

Reply:
> Do DipStrike and FaceAligned actually mean something different?

Reply:
Okay